frontend/src/App.js

```diff
@@ -159,8 +159,8 @@ const App = () => {
             const role_details = config.roles.find(
               (r) => r.id === res.data.role.id
             );
-            const designation = config?.designations.find(
-              (d) => d.id === res.data.role.id
```

Comment:
and please read the issue carefully @lightme-fan
https://github.com/akvo/rtmis/issues/496